docs/faq/import-beast.md

```diff
@@ -66,4 +66,8 @@ augur import beast --mcc data/beast.mcc.nex --output-tree results/mers.new
     --most-recent-tip-date 2018.43
 ```
 
-A full [example build can be found here](https://github.com/nextstrain/augur/tree/master/tests/builds/beast_mers).
+A full [example build can be found here][]. (N.b., this build has been
+removed from the current version of the repository; the link here is
+to the last version prior to its removal from the repo.)
+
+[example build can be found here]: ](https://github.com/nextstrain/augur/blob/73b8a71103bae3a36b49acdee7c7c75a7e69a751/tests/builds/beast_mers/Snakefile
```

Comment:
If you want to be pedantic, a better link for "last version prior to its removal from the repo" would be https://github.com/nextstrain/augur/tree/fa9bedaa9a7b6b6a40fc11a28bc4f68b28b890a9~/tests/builds/beast_mers since fa9bedaa9a7b6b6a40fc11a28bc4f68b28b890a9 is the commit that removed it. But there aren't any changes to this folder between that link and yours, so practically what you have is just fine.